homeassistant/components/tautulli/strings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "To find your API key, open the Tautulli webpage and navigate to Settings and then to Web interface. The API key will be at the bottom of that page.\n\nExample of the URL: ```http://192.168.0.10:8181``` with 8181 being the default port.",
```

Comment:
URLs and similar things that shouldn't be translated are preferably set in `description_placeholders` and not directly in the translation string.